README.md

```diff
@@ -93,20 +93,46 @@ nyc report --reporter=lcov
 
 ## Excluding Files
 
-By default nyc does not instrument files in `node_modules`, or `test`
-for coverage. You can override this setting in your package.json, by
-adding the following configuration:
+You can tell nyc to exclude specific files and directories by adding
+an `config.nyc.exclude` array to your `package.json`. Each element of
+the array is a glob pattern indicating which paths should be omitted.
 
-```js
+Globs are matched using [micromatch](https://www.npmjs.com/package/micromatch)
+
+In addition to patterns specified in the package, nyc will always exclude
+files in `node_modules`.
+
+For example, the following config will exclude all `node_modules`,
+any files with the extension `.spec.js`, and anything in the `build`
+directory:
+
+```json
 {"config": {
   "nyc": {
     "exclude": [
-      "node_modules/"
+      "**/*.spec.js",
+      "build"
     ]
   }
 }}
 ```
 
+> Note: exclude defaults to `['test', 'test{,-*}.js']`, which would exclude
```

Comment:
There's a note paragraph at the end of the readme which is in italics, yet this is a quote. Would be nice to be consistent.